### ISSUE TITLE
[CRASH] ValueClone: Unable to clone value on 1.1.8

### ISSUE THREAD

**Issue by kravetsone** [elysiajs/elysia#796]
### What version of Elysia is running?

1.1.8

### What platform is your computer?

Microsoft Windows NT 10.0.22631.0 x64

### What steps can reproduce the bug?

![image](https://github.com/user-attachments/assets/474ee107-e750-496b-8906-b325d58df08f)

**Comment by kravetsone** [comment]
Workaround is patch typebox Clone function

```patch
diff --git a/build/esm/value/clone/clone.mjs b/build/esm/value/clone/clone.mjs
old mode 100644
new mode 100755
index 0611003e6ea2360aeed7fcab74e3c72c75dd48e3..2434d92f5c82c8d68d5e14932ff775685a07f8f8
--- a/build/esm/value/clone/clone.mjs
+++ b/build/esm/value/clone/clone.mjs
@@ -42,5 +42,7 @@ export function Clone(value) {
         return TypedArrayType(value);
     if (IsValueType(value))
         return ValueType(value);
-    throw new Error('ValueClone: Unable to clone value');
+    
+    return structuredClone(value);
+    // throw new Error('ValueClone: Unable to clone value');
 }

```

It's solve this error but maybe not so good

**Comment by SaltyAom** [comment]
Hi, thanks for the report.
Would be awesome if you can also provide a reproducible example code so I can replicate it on my end and fix the problem. Thanks.

**Comment by kravetsone** [comment]
> Hi, thanks for the report. Would be awesome if you can also provide a reproducible example code so I can replicate it on my end and fix the problem. Thanks.

Okay i will do it) it crashed my production application so I was desperately looking for a solution
Now i can provide it here

**Comment by kravetsone** [comment]
```sh
} UNKNOWN 42 |         return TypedArrayType(value);
43 |     if (IsValueType(value))
44 |         return ValueType(value);
45 |
46 |     // return structuredClone(value);
47 |     throw new Error('ValueClone: Unable to clone value');
               ^
error: ValueClone: Unable to clone value
      at Clone (Z:\PROJECTS\PAY\catch-up\node_modules\@sinclair\typebox\build\esm\value\clone\clone.mjs:47:11)
      at Z:\PROJECTS\PAY\catch-up\node_modules\@sinclair\typebox\build\esm\value\clean\clean.mjs:32:82
      at map (1:11)
      at FromIntersect (Z:\PROJECTS\PAY\catch-up\node_modules\@sinclair\typebox\build\esm\value\clean\clean.mjs:32:40)  
      at K (Z:\PROJECTS\PAY\catch-up\node_modules\elysia\src\utils.ts:712:23)
```

**Comment by kravetsone** [comment]
well i reproduce it

**Comment by kravetsone** [comment]
It's maybe typebox bug too

```ts
import Elysia, { t } from "elysia";

export const userSchema = t.Intersect([
	t.Object({ some: t.String() }),
	t.Object({
		socialLinks: t.String({}),
	}),
]);

const app = new Elysia().get("/test", () => ({}), {
	response: userSchema,
});

const response = await app.handle(
	new Request("http://localhost/test", {
		method: "GET",
	}),
);
console.log(response, await response.text());
```

**Comment by SaltyAom** [comment]
Should have been fixed with 68987fa

**Comment by kravetsone** [comment]
```sh
PS Z:\PROJECTS\reproduction> bun .\index.ts    
Response (0 KB) {
  ok: false,
  url: "",
  status: 500,
  statusText: "",
  headers: Headers {},
  redirected: false,
  bodyUsed: true
} {"name":"Error","message":"ValueClone: Unable to clone value"}
PS Z:\PROJECTS\reproduction> bun update elysia --latest
bun update v1.1.26 (0a37423b)

installed elysia@1.1.11

1 package installed [761.00ms]
PS Z:\PROJECTS\reproduction> bun .\index.ts
Response (0 KB) {
  ok: false,
  url: "",
  status: 422,
  statusText: "",
  headers: Headers {},
  redirected: false,
  bodyUsed: true
} {
  "type": "validation",
  "on": "response",
  "summary": "Expected all values to match",
  "property": "root",
  "message": "Expected all values to match",
  "expected": {
    "some": "",
    "socialLinks": ""
  },
  "found": {},
  "errors": [
    {
      "summary": "Expected all values to match",
      "type": 29,
      "schema": {
        "type": "object",
        "allOf": [
          {
            "type": "object",
            "properties": {
              "some": {
                "type": "string"
              }
            },
            "required": [
              "some"
            ]
          },
          {
            "type": "object",
            "properties": {
              "socialLinks": {
                "type": "string"
              }
            },
            "required": [
              "socialLinks"
            ]
          }
        ],
        "additionalProperties": false
      },
      "path": "",
      "value": {},
      "message": "Expected all values to match"
    },
    {
      "type": 45,
      "schema": {
        "type": "string"
      },
      "path": "/some",
      "message": "Required property",
      "summary": "Property 'some' is missing"
    },
    {
      "summary": "Expected all values to match",
      "type": 29,
      "schema": {
        "type": "object",
        "allOf": [
          {
            "type": "object",
            "properties": {
              "some": {
                "type": "string"
              }
            },
            "required": [
              "some"
            ]
          },
          {
            "type": "object",
            "properties": {
              "socialLinks": {
                "type": "string"
              }
            },
            "required": [
              "socialLinks"
            ]
          }
        ],
        "additionalProperties": false
      },
      "path": "",
      "value": {},
      "message": "Expected all values to match"
    },
    {
      "type": 45,
      "schema": {
        "type": "string"
      },
      "path": "/socialLinks",
      "message": "Required property",
      "summary": "Property 'socialLinks' is missing"
    }
  ]
}
```

It works for me, thanks!
